runners/flink/src/test/java/org/apache/beam/runners/flink/streaming/UnboundedSourceWrapperTest.java

```diff
@@ -277,7 +281,7 @@ public void collect(
                         public void close() {}
                       });
                 } catch (Exception e) {
-                  System.out.println("Caught exception: " + e);
+                  LOG.info("Caught exception:", e);
```

Comment:
Would log with error level.

Reply:
Since this exception is handled, there is strictly no logging needed here at all.

Reply:
This is the same as the exception above. It is an unexpected error. But we assume that the assertions following won't pass. I'm ok with keeping it at INFO level.